### ISSUE TITLE
Extract Minter into an independent service

### ISSUE THREAD

**Issue by eecavanna** [microbiomedata/nmdc-runtime#484]
Some database migrations will involve creating new documents and inserting those documents into Mongo collections. The NMDC schema requires all documents in those collections to have minted IDs in one of their fields. Since the migration process is not atomic (i.e. during the process, the database temporarily does not necessarily adhere to any version of the NMDC schema), we generally do not want to leave the Runtime API online during the migration.

Today, the minter is part of the Runtime API. This ticket represents the task of extracting the minter from the Runtime API and create an independent service that just does minting.

> Note: The initial issue description is being written by someone that has never used the minter before.

Related: https://github.com/microbiomedata/nmdc-schema/issues/1675

CC: @dwinston

**Comment by eecavanna** [comment]
Side note: Since (I think) the database collection used by the minter is not represented in the NMDC schema, maybe that collection could be extracted into a separate database also. I don't know yet what the pros and cons of that would be (haven't thought about it).

**Comment by dwinston** [comment]
yes, I think the extraction would include the database collection(s).

**Comment by PeopleMakeCulture** [comment]
@eecavanna Are we good to close this ticket?

**Comment by eecavanna** [comment]
Hi @PeopleMakeCulture, I'd prefer that it be left open and moved to the backlog. I am afraid that, if we close it, people will think it has been completed (as opposed to "postponed"). I will move it to the backlog now.

**Comment by eecavanna** [comment]
Adding this note for future reference:

Late last week, the migration squad members developing migrations decided they would pursue the option of minting `WorkflowChain` IDs _before_ running migrations, using an existing, _non-standalone_ Minter. That is instead of having the migrator access a standalone Minter during the migration process (this ticket was about setting up that standalone Minter).

I still expect the Minter to eventually be a standalone service, but I think pre-minting is more practical at this point, given that people want to be able to explore the migrated data ASAP.
